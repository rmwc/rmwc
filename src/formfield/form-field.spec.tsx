import React from 'react';
import { render } from '@testing-library/react';
import { FormField } from './';

describe('FormField', () => {
  test('renders', () => {
    const { asFragment } = render(
      <FormField>
        <input type="checkbox" id="input" />
        <label htmlFor="input">Input Label</label>
      </FormField>
    );
    expect(asFragment()).toMatchSnapshot();
  });
});
