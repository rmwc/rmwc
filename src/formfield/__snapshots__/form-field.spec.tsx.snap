// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FormField renders 1`] = `
<DocumentFragment>
  <div
    class="mdc-form-field"
  >
    <input
      id="input"
      type="checkbox"
    />
    <label
      for="input"
    >
      Input Label
    </label>
  </div>
</DocumentFragment>
`;
